doc/manuals/admin/cli.md

```diff
@@ -115,13 +115,15 @@ The list of available options is the following:
     for REST connections. Note that the default value is zero, i.e., no timeout (wait forever).
 -   **-cprForwardLimit**. Maximum number of forwarded requests to Context Providers for a single client request
     (default is no limit). Use 0 to disable Context Providers forwarding completely.
--   **-corsOrigin <domain>**. Configures CORS for GET requests,
+-   **-corsOrigin <domain>**. Enables Cross-Origin Resource Sharing,
```

Comment:
Maybe you could add a sentence with a link to the nice document about CORS you mentioned, e.g.

> Please find more information about CORS in [this link](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS).

Reply:
Fixed in f97a6f1